core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPageByteUtil.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.carbondata.core.datastore.page;
+
+/**
+ * Utility methods to converts to primitive types only column page data decode.
+ */
+public class ColumnPageByteUtil {
```

Comment:
I remember there is another `ByteUtil` in core module, can we use that instead of duplicating code

Reply:
its not duplicating, this conversion is in Little_endian so kept here, anyway this also moved to ByteUtil